dev/io.openliberty.microprofile.health.3.1.internal/bnd.bnd

```diff
@@ -0,0 +1,101 @@
+
+Import-Package: \
+  !com.ibm.ws.microprofile.health.spi.impl, \
+  !com.ibm.ws.microprofile.health.impl,\
+  javax.enterprise.context.spi; version="[1.1,3)",\
+  javax.enterprise.inject.*; version="[1.1,3)",\
+  javax.enterprise.util; version="[1.1,3)",\
+  *
+
+Export-Package: \
+	!io.openliberty.microprofile.health31.resources,\
+	io.openliberty.microprofile.health30.impl,\
+	io.openliberty.microprofile.health30.internal,\
+	io.openliberty.microprofile.health30.services,\
+	io.openliberty.microprofile.health31.spi.impl;version=1.0
+
+Private-Package: \
+	com.ibm.ws.microprofile.health.internal,\
+	com.ibm.ws.microprofile.health.services,\
+	com.ibm.ws.microprofile.health.services.impl,\
+	com.ibm.ws.microprofile.health.util,\
+	io.openliberty.microprofile.health31.*
+
+Include-Resource: \
+    META-INF/services=resources/META-INF/services, \
+    WEB-INF=resources/WEB-INF
+
+
+src: src,resources
+testsrc: test/src
+
+-buildpath: \
+        com.ibm.websphere.javaee.annotation.1.3;version=latest,\
+        com.ibm.websphere.javaee.servlet.4.0; version=latest,\
+        com.ibm.websphere.javaee.cdi.2.0;version=latest,\
+        com.ibm.websphere.jsonsupport;version=latest,\
+        io.openliberty.org.eclipse.microprofile.health.3.1;version=latest,\
+        io.openliberty.org.eclipse.microprofile.config.2.0;version=latest,\
+        com.ibm.websphere.org.osgi.core;version=latest,\
+        com.ibm.websphere.org.osgi.service.component;version=latest,\
+        com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+        com.ibm.ws.cdi.interfaces;version=latest,\
+        com.ibm.ws.classloader.context;version=latest,\
+        com.ibm.ws.classloading;version=latest,\
+        com.ibm.ws.container.service.compat;version=latest,\
+        com.ibm.ws.container.service;version=latest,\
+        com.ibm.ws.context;version=latest,\
+        com.ibm.ws.javaee.metadata.context;version=latest,\
+        com.ibm.ws.javaee.version;version=latest,\
+        com.ibm.ws.kernel.service;version=latest,\
+        com.ibm.ws.logging;version=latest,\
+        com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+        com.ibm.ws.threading;version=latest,\
+        com.ibm.ws.webcontainer;version=latest,\
+        com.ibm.ws.org.apache.commons.lang3;version=latest,\
+        org.eclipse.osgi;version=latest,\
+        com.ibm.ws.microprofile.health;version=latest,\
+        io.openliberty.microprofile.health.3.0.internal;version=latest
+
+-testpath: \
+        ../build.sharedResources/lib/junit/old/junit.jar;version=file, \
+        com.ibm.ws.junit.extensions;version=latest, \
+        com.ibm.websphere.javaee.jsonp.1.0;version=latest, \
+        org.hamcrest:hamcrest-all;version=1.3, \
+        org.jmock:jmock-junit4;version=2.5.1, \
+        org.jmock:jmock;version=2.5.1, \
+        org.glassfish:javax.json;version=1.0.4, \
+        org.slf4j:slf4j-api;version=1.7.7, \
+        org.slf4j:slf4j-jdk14;version=1.7.7, \
```

Comment:
Please use the common bundles instead;
```suggestion
        com.ibm.ws.org.slf4j.api.1.7.7;version=latest, \
        com.ibm.ws.org.slf4j.jdk14.1.7.7;version=latest, \
```
We ought to remove the version number from the bundle name, but that is a separate problem

Reply:
Updated for both 3.0 and 3.1